tools/packaging/snapshotter-deploy/nydus-snapshotter.yaml

```diff
@@ -0,0 +1,141 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nydus-snapshotter
+  labels:
+    app: nydus-snapshotter
+  namespace: nydus-snapshotter
+data:
+  config-coco-guest-pulling.toml: |-
+    version = 1
+    # Snapshotter's own home directory where it stores and creates necessary resources
+    root = "/var/lib/containerd-nydus"
+    # The snapshotter's GRPC server socket, containerd will connect to plugin on this socket
+    address = "/run/containerd-nydus/containerd-nydus-grpc.sock"
+    [daemon]
+    # Enable proxy mode
+    fs_driver = "proxy"
+    [snapshot]
+    # Insert Kata volume information to `Mount.Options`
+    enable_kata_volume = true
+  
+  
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nydus-snapshotter
+  namespace: nydus-snapshotter
+  labels:
+    app: nydus-snapshotter
+spec:
+  selector:
+    matchLabels:
+      app: nydus-snapshotter
+  template:
+    metadata:
+      labels:
+        app: nydus-snapshotter
+
+    spec:
+      serviceAccountName: nydus-snapshotter-sa
+      hostNetwork: true
+      hostPID: true
+      containers:
+      - name: nydus-snapshotter
+        image: "ghcr.io/containerd/nydus-snapshotter:latest"
+        imagePullPolicy: Always
+        lifecycle:
+          postStart:
+            exec:
+              command:
+              - "/bin/sh"
+              - "-c"
+              - |
+                # currently, without host pid in container, we can not nsenter with pid and can not invoke systemctl correctly.
+                nsenter -t 1 -m systemctl -- restart containerd
+        command: 
+         - /bin/sh
+         - -cx
+         - |-
+           nydus_snapshotter_config=/etc/nydus/config-coco-guest-pulling.toml
+           /usr/local/bin/containerd-nydus-grpc --config ${nydus_snapshotter_config} --log-to-stdout
+        volumeMounts:
+        - name: config
+          mountPath: "/etc/nydus/"
+        - name: nydus-lib
+          mountPath: "/var/lib/containerd-nydus"
+          mountPropagation: Bidirectional
+        - name: nydus-run
+          mountPath: "/run/containerd-nydus"
+          mountPropagation: Bidirectional
+
+        securityContext:
+          privileged: true
+
+      initContainers:
+      - name: update-containerd
+        image: "ghcr.io/liubin/toml-cli:v0.0.7"
```

Comment:
ghcr.io/liubin/toml-cli:v0.0.7 seems is amd64 arch only, so after the script `deploy_nydus_snapshotter` function, 
I got the pod in CrashLoopBackOff status.
```
nydus-snapshotter   nydus-snapshotter-g78x5                        0/1     Init:CrashLoopBackOff   5 (17s ago)   3m35s
```
Can we change the image to mutli-arch image?

Reply:
Fixed.